### DISCUSSION TITLE
Social Media

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [FossifyOrg/General-Discussion#12] [Ideas]
we need socials, last one linked them all in the social tab but you can see how thwyre nkt foss ones. it'd be good to habe a matrix server or a revolt https://github.com/revoltchat?ref=its-foss and remove the reddit and make a lemmy communitt instead. mastodon account would be cool too.

**Comment by naveensingh** [comment]
I believe raising awareness about Fossify (and FOSS in general) can be done more effectively using channels people already use rather than using channels that themselves are FOSS.

We'll have Reddit, Telegram, and some of your suggestions. I don't use Facebook, I never liked it.

**Reply by inson1** [reply]
I think Facebook is dead or at least is on the edge of death.

**Reply by ghost** [reply]
Twitter is also very much dead, I would suggest Mastodon instead of the Facebook Owned Threads or Bluesky Created by the Twitter Founder.
I would avoid making a discord server a primary location (Due to limits on how many people can join, inability for information to get out [Via Search Engine for example], along with discord being proprietary) though It's not something that can't be done at all.

**Reply by inson1** [reply]
Mastodon > Bluesky

**Reply by inson1** [reply]
Mastodon, Bluesky, Twitter (X) is for different type of communation than Reddit, Matrix, Lemmy etc. - "admin based" vs "community based" and it would take longer time with less benefit for naveensingh, so maybe would be better to just work on new features/bugs

**Reply by inson1** [reply]
Or maybe use some platform for copying posts and then have all important platforms

**Reply by NamelessGO** [reply]
TL:DR We should set discord as a base of the community but allow a bridge with matrix

Inability to be in the search engine, is not really a problem because people who uses FOSS apps generally go to source code or the docs for an overview of the applicatio, and in it, will find the discord link invite

"Discord is proprietary, we shouldn't use it"

Unfortunately, discord it's easier to navigate and it is more popular, but we should allow matrix users to "join" discord by making a bridge between the applications

------
Revolt is nice too, but we shouldn't exclude the users from discord, is there a way to bridge revolt?

**Reply by NamelessGO** [reply]
Telegram is fine, but I think most of us uses it for piracy than communities haha

Edit: Telegram has great communities features, disregard the joke in the paragraph above

**Reply by ghost** [reply]
> Inability to be in the search engine, is not really a problem because people who uses FOSS apps generally go to source code or the docs for an overview of the applicatio, and in it, will find the discord link invite
My issue with Discord is, it seems at least from what I've notice, alot of "Go check discord" exists, which is not very helpful if Mom, Dad, Grandpa, whoever you might have installed these apps for is looking for help and doesn't want to disturb you, or even yourself if you can't, or don't want to use a discord. That's my concern with that.

To be Honest, I personally don't really use Discord anymore, and I haven't found a Matrix client that works for me and has multiple user Support (I have specific reasons for a Private, De-federated Matrix Server)

> Telegram is fine, but I think most of us uses it for piracy than communities haha
Honestly, I rarely even heard of Telegram, let alone ever used it at all.

**Reply by inson1** [reply]
@NameLessGO I have to ask. Why for piracy? I dont get it.

**Reply by NamelessGO** [reply]
> @NameLessGO I have to ask. Why for piracy? I dont get it.

Just something that telegram is famous here

Changed my stance after experienced the WhatsApp community feature 

If you want to create a community there, it would be great, telegram is awesome for communities

**Comment by inson1** [comment]
I wouldnt remove reddit, but I would add lemmy and matrix.

**Reply by naveensingh** [reply]
sure :)

**Reply by nehemiagurl** [reply]
Very much in favour of adding Lemmy. And since Lemmy is a Fediverse app, you get a Mastodon account for free! (well not really a Mastodon account but people can follow you from Mastodon which is what Mastodon users care about)

**Reply by realeatham** [reply]
Don't make the lemmy on lemmy.world, it is too popular and things should be spread out in the fediverse as much as possible to prevent one person from taking control. Make it on a smaller instance.

**Comment by jancordeiro** [comment]
I can develop a simple website for the project using HTML and CSS only. Without JS of course... can I make this contribution?

**Reply by inson1** [reply]
https://github.com/FossifyOrg/fossifyorg.github.io/issues/1

I think we will be using this tech stack :)

> Use Astro + Preact Islands for Interactivity + Open-Props for CSS to build the website.

**Reply by RustoMCSpit** [reply]
why without js?

**Reply by inson1** [reply]
@RustoMCSpit maybe he just didnt learn it yet or part of privacy community doesnt like js

**Reply by ghost** [reply]
There is also the fact some browsers or extensions auto block all JS, even from the primary domain. It would be ideal if the website worked at least on a basic level for those users as well. I find it really really annoying when I visit a website and get a white screen or a broken site because everything relies on JS, then I have to figure out which script domain is actually the important one if I wanna use it.

**Reply by inson1** [reply]
@hayleethegamer which extension?

**Reply by nehemiagurl** [reply]
@inson1 there's a lot, depending on the config. iirc both Privacy Badger and NoScript have configs that break 1st party JS, and of course uBlock Origin can be made to block virtually anything.
there's also browsers that have built-in JS blocks, like IceCat, Mullvad and of course Tor.

**Reply by ghost** [reply]
@inson1 
> @hayleethegamer which extension?

Multiple of them

I use Firefox (Which has messed with websites before with only it's built in protections), NoScript, UBlock, and Privacy Badger. Most of the time I find the culprit of First Party Blocks is NoScript doing it's job.
The Main probably really starts when a website relies on a dozen external websites for scripts and it's unclear which are for functionality, and which are for Ads, Tracking, or just Non-sense that adds nothing and eats Bandwidth.

**Reply by basings** [reply]
@inson1 

Here's some food 
https://www.gnu.org/philosophy/javascript-trap.html
https://www.fsf.org/campaigns/freejs/
https://www.gnu.org/software/librejs/

**Comment by nehemiagurl** [comment]
I think it's important to have multiple social accounts even for crossposting, but it's also super important to make sure there's an official, open-web channel for discussions and announcements. We just saw how annoying it can be when the devs only announce something on, say, Reddit.
The ideal would be maintaining interactions on all platforms. But this is well and truly unfeasible even for much larger projects than Fossify. So the second best option is to go the [Framasoft](https://framapiaf.org/@framasoft) way - have a Twitter account, a Mastodon/Lemmy account, have a Matrix chat, but make it very clear that those accounts are not meant for support purposes, and that support is done in the repo. Rather those accounts are crossposting each other for wider reach and engagement.
Important announcements (which hopefully won't include stuff like "we're selling to ZipoApps" but you know there's always important stuff to annouce) should be posted to all platforms, as well as the repo and the website. That way you're not leaving anyone out.

**Reply by Korb** [reply]
I agree that posting only on **Reddit** (for example) is a bad idea. Because all I see on that site when using a VPN is this image:

<img width="50%" height="50%" src="https://github.com/user-attachments/assets/12298875-bdb4-406a-859f-71017434d6d0" />

It's funny that I can only challenge the blocking by creating a topic, which I can't create due to lack of access. Without using a VPN, I see the splash screen of my national censor, and I can't buy another VPN that doesn't leak data to the mafia that currently controls my country, because the country is disconnected by sanctions and boycotts from international payment systems. As well as from Facebook, X, Instagram and several thousand other sites and services.

**Telegram** could not be blocked, but in light of the news about its data centers, I no longer consider it the best **popular** alternative.

**Comment by rafnov** [comment]
Everything but Discord. It's horrible. 
There is already Telegram channel but closed for announcements and not active. I would use what most FOSS uses. Matrix my candidate.

**Reply by realeatham** [reply]
Xmpp is also a nice Foss chat platform

**Comment by MusicNote-Haven** [comment]
I have worked for a project "Presearch" and I am still actively using this search engine. I know the investors and can talk with them about Fossify, you think this is a good idea? Maybe work together? They are also off-Google minded (like me, and I also don't use Windows anymore). You have so many Apps that combining a search engine is not a bad idea? They also have a marketing department. This is the link with more information https://presearch.com/learn-more

**Reply by naveensingh** [reply]
A search engine does not fit into what Fossify does now. Maybe someday in the future.

**Reply by MusicNote-Haven** [reply]
Ok, just let me know, I can help bring more awareness
